### ISSUE TITLE
Feature Request: Lax mode

### ISSUE THREAD

**Issue by wSedlacek** [lukastaegert/eslint-plugin-tree-shaking#81]
Right now this plugin seems a little too aggressive for my needs.

Let's look at this small example.

<img width="591" alt="Screen Shot 2021-03-04 at 7 58 03 PM" src="https://user-images.githubusercontent.com/8206108/110060944-f3a5f880-7d23-11eb-91be-648b8a009cfa.png">

I am using RxJS that from my understanding is pretty good about tree shaking, and even if they aren't there isn't much I can do from my project to fix their problems and I would rather not have to add a bunch of eslint-disable everywhere.

So I am looking for options that will assume that other modules do not have any side effects.
I just want to make sure that the current module doesn't create any new side effects.

I also might be ignorant of this but does this
```
export class DeckService {
  private readonly internalDecks$ = new BehaviorSubject<Deck[]>([]);
  public readonly decks$: Observable<Deck[]>;

  constructor() {
    this.decks$ = this.internalDecks$.asObservable();
  }
}
```

Really differ from this?
```
export class DeckService {
  private readonly internalDecks$ = new BehaviorSubject<Deck[]>([]);
  public readonly decks$ = this.internalDecks$.asObservable();
}
```

Or is this just a false positive?

**Comment by wSedlacek** [comment]
Duplicate of #73
